vantage6/node/docker/utils.py

```diff
@@ -8,6 +9,17 @@
 log = logging.getLogger(logger_name(__name__))
 
 
+def is_docker_running() -> bool:
+    """ Return True if docker engine is running"""
+    try:
+        client = docker.from_env()
+        client.ping()
+    except Exception as e:
```

Comment:
I think catching `docker.errors.APIError` is more appropriate , see [here](https://docker-py.readthedocs.io/en/stable/client.html?highlight=ping#docker.client.DockerClient.ping)